subsys/bluetooth/rpc/host/bt_rpc_conn_host.c

```diff
@@ -1287,6 +1287,12 @@ void bt_rpc_auth_cb_pairing_failed(struct bt_conn *conn, enum bt_security_err re
 
 static int bt_conn_auth_cb_register_on_remote(uint16_t flags)
 {
+	if (flags & FLAG_AUTH_CB_IS_NULL) {
+
```

Comment:
Blank lines not needed (this one and line 1293)

Reply:
Fixed.